.github/workflows/deploy-docs.yml

```diff
@@ -11,26 +11,15 @@ jobs:
     environment: production
     steps:
       - uses: actions/checkout@v2
-        with:
-          token: ${{ secrets.CURSORLESS_BOT_TOKEN }}
-      - name: Configure GPG Key
-        run: |
-          echo -n "$GPG_SIGNING_KEY" | base64 --decode | gpg --import
-        env:
-          GPG_SIGNING_KEY: ${{ secrets.CURSORLESS_BOT_GPG_SIGNING_KEY }}
-      - name: git config
-        run: |
-          git config user.name cursorless-bot
-          git config user.email 98099035+cursorless-bot@users.noreply.github.com
-          git config user.signingkey A9387720AFC62221
-          git config commit.gpgsign true
```

Comment:
So looking at the commit to `gh-pages`, it appears all this stuff was simply ignored 😅.  I think just using the token and username stuff below should give us the protection we need.